src/app/modules/account/services/msp-api-account.service.ts

```diff
@@ -1427,8 +1427,17 @@ export class MspApiAccountService extends AbstractHttpService {
       .addressUpdate
       ? 'Y'
       : 'N';
-    accountHolder.selectedPersonalInfoChange = from.accountChangeOptions.personInfoUpdate ? 'Y' : 'N';
-    accountHolder.selectedStatusChange = from.applicant.updateStatusInCanada ? 'Y' : 'N';
+    accountHolder.selectedPersonalInfoChange = (from.accountChangeOptions
+      .personInfoUpdate || from.applicant.updatingPersonalInfo === true)
+      ? 'Y'
+      : 'N';
+    accountHolder.selectedStatusChange =
+      from.applicant.updateStatusInCanada ||
+      (from.addedSpouse && from.addedSpouse.updateStatusInCanada) ||
+      (from.removedSpouse && from.removedSpouse.updateStatusInCanada) ||
+      (from.updatedSpouse && from.updatedSpouse.updateStatusInCanada)
+        ? 'Y'
+        : 'N';
```

Comment:
this is what was causing DEAM-318 (Spouse updates should not affect selectedStatusChange)